content/docs/addons-test-utils.md

```diff
@@ -103,28 +103,28 @@ afterEach(() => {
   container = null;
 });
 
-it('can render and update a counter', () => {
+it('काउंटर रेंडर और अपडेट कर सकते हैं', () => {
   // Test first render and componentDidMount
   act(() => {
     ReactDOM.render(<Counter />, container);
   });
   const button = container.querySelector('button');
   const label = container.querySelector('p');
-  expect(label.textContent).toBe('You clicked 0 times');
-  expect(document.title).toBe('You clicked 0 times');
+  expect(label.textContent).toBe('आपने 0 बार क्लिक किया');
+  expect(document.title).toBe('आपने 0 बार क्लिक किया');
 
   // Test second render and componentDidUpdate
   act(() => {
     button.dispatchEvent(new MouseEvent('click', {bubbles: true}));
   });
-  expect(label.textContent).toBe('You clicked 1 times');
-  expect(document.title).toBe('You clicked 1 times');
+  expect(label.textContent).toBe('आपने 1 बार क्लिक किया');
+  expect(document.title).toBe('आपने 1 बार क्लिक किया');
 });
 ```
 
-- Don't forget that dispatching DOM events only works when the DOM container is added to the `document`. You can use a library like [React Testing Library](https://testing-library.com/react) to reduce the boilerplate code.
+- यह मत भूलिए कि DOM इवेंट्स भेजना सिर्फ़ तभी काम करता है जब DOM कंटेनर को `document` में ऐड किया जाता है। आप बॉयलरप्लेट कोड को कम करने के लिए [React Testing Library](https://testing-library.com/react) जैसी लाइब्रेरी का उपयोग कर सकते हैं।
 
-- The [`recipes`](/docs/testing-recipes.html) document contains more details on how `act()` behaves, with examples and usage.
+- [`रेसिपी`](/docs/testing-recipes.html) डॉक्यूमेंट में उदाहरण और उपयोग के साथ `act()` कैसे व्यवहार करता है, इस पर अधिक विवरण है।
```

Comment:
कैसे व्यवहार करता है could be replaced with a better translation

Reply:
Unable to think of better word...

Reply:
बर्ताव?

Reply:
Done